packages/xrpl/test/models/utils.test.ts

```diff
@@ -28,6 +30,40 @@ import {
  * Provides tests for utils used in models.
  */
 describe('Models Utils', function () {
+  describe('validate containsDuplicates utility method', function () {
+    it(`use nested-objects for input parameters`, function () {
+      const list_without_dups: AuthorizeCredential[] = [
+        { Credential: { Issuer: 'alice', CredentialType: 'Passport' } },
+        { Credential: { CredentialType: 'DMV_license', Issuer: 'bob' } },
+      ]
+
+      // changing the order of the inner-objects in the list
+      const list_with_duplicates: AuthorizeCredential[] = [
+        { Credential: { Issuer: 'alice', CredentialType: 'Passport' } },
+        { Credential: { CredentialType: 'Passport', Issuer: 'alice' } },
+      ]
+
+      assert.isFalse(containsDuplicates(list_without_dups))
+      assert.isTrue(containsDuplicates(list_with_duplicates))
```

Comment:
These should be separate tests

Reply:
[2080c21](https://github.com/XRPLF/xrpl.js/pull/2874/commits/2080c21bd425a596f844d2898ed65c465f19adfd)